Sources/MacroToolkit/ClassRestrictionType.swift

```diff
@@ -5,7 +5,8 @@ public struct ClassRestrictionType: TypeProtocol {
     public var _baseSyntax: ClassRestrictionTypeSyntax
     public var _attributedSyntax: AttributedTypeSyntax?
 
-    public init(_ syntax: ClassRestrictionTypeSyntax, attributedSyntax: AttributedTypeSyntax? = nil) {
+    public init(_ syntax: ClassRestrictionTypeSyntax, attributedSyntax: AttributedTypeSyntax? = nil)
```

Comment:
Wrap these arguments onto newlines instead of wrapping the brace (which I assume that swift format did 😅),

```swift
public init(
    _ syntax: ClassRestrictionTypeSyntax,
    attributedSyntax: AttributedTypeSyntax? = nil
) {
```